packages/runner/package.json

```diff
@@ -7,10 +7,9 @@
   "type": "module",
   "scripts": {
     "start": "runner",
-    "prepare": "(cd frontend && npm install); (cd test_app/frontend && npm install);",
+    "prepare": "cd frontend && npm install",
     "lint": "eslint . --ignore-path .gitignore --ext .js",
-    "build": "concurrently \"cd frontend && npm run build\"",
-    "dev": "runner -Dde --dev"
+    "build": "cd frontend && npm run build"
```

Comment:
notice that all of our scripts here become simpler when pulling out the `test-app` package. We no longer need `concurrently` dependency. Our `runner` package becomes smaller.